examples/fluids/README.md

```diff
@@ -107,16 +146,6 @@ For the 2D advection problem, the following additional command-line options are
   - Default value
   - Unit
 
-* - `-lx`
-  - Length scale in x direction
-  - `8000`
-  - `m`
-
-* - `-ly`
-  - Length scale in y direction
-  - `8000`
-  - `m`
-
```

Comment:
Somewhere, can we be explicit that lengths like `-dm_plex_box_upper` are given in meters, and will be nondimensionalized according to `-units_meter`?

Reply:
Good point. If the last commit looks good, I can squash the last two commits and merge.

Reply:
Docs look good. Have you noticed the failing HIP tests?

Reply:
Sorry I missed [your comment](https://github.com/CEED/libCEED/pull/881#issuecomment-1027434984). It seems that HIP is not happy about excluding the outflow QFunction from `eulervortex.h`. I am not sure about the reason though!

Reply:
I think it's better not to duplicate the struct definition anyway so I think it's fine to merge. We can find the root cause if we have a good reason for splitting these headers in the future.